src/main/bash/envs/kobman-kobman.sh

```diff
@@ -2,12 +2,12 @@
 function __kobman_development_kobman_dir {
 
```

Comment:
Why do we need a seperate dir function? what does the name signify?

Reply:
This function was created as part of the requirement in which the environment specific folders has to created @ home directory (outside .kobman)